connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -224,85 +225,64 @@ public void tearDown() {
         if (metrics != null) {
             metrics.stop();
         }
+
     }
 
     @Test
     public void testSinkTasksCloseErrorReporters() throws Exception {
-        ErrorReporter reporter = EasyMock.mock(ErrorReporter.class);
+        ErrorReporter reporter = mock(ErrorReporter.class);
 
         RetryWithToleranceOperator retryWithToleranceOperator = operator();
         retryWithToleranceOperator.reporters(singletonList(reporter));
 
         createSinkTask(initialState, retryWithToleranceOperator);
 
-        expectInitializeTask();
```

Comment:
We are missing some verification for calls which were earlier in this function.

I would suggest to create a method called `verifyInitializeTask()` which would verify the following invocations:
1. `verify(sinkTask).start(TASK_PROPS);`
2. `verify(sinkTask).initialize(any(WorkerSinkTaskContext.class));`
3. `verify(consumer).subscribe(singletonList(TOPIC), rebalanceListener);`

You can use the method to verify initialize calls in all tests that were earlier using `expectInitializeTask()` earlier.